test/common/crudtester.go

```diff
@@ -714,27 +715,133 @@ func (c *FederatedTypeCrudTester) versionForCluster(version *fedv1a1.PropagatedV
 	return ""
 }
 
+func (c *FederatedTypeCrudTester) CheckRemoteStatus(fedObject *unstructured.Unstructured, targetObject *unstructured.Unstructured) {
+	for clusterName := range c.testClusters {
+		clusterConfig := c.testClusters[clusterName].Config
+
+		kubeClient := kubeclientset.NewForConfigOrDie(clusterConfig)
+		WaitForNamespaceOrDie(c.tl, kubeClient, clusterName, targetObject.GetNamespace(),
+			c.waitInterval, 30*time.Second)
+
+		util.AddManagedLabel(targetObject)
+		labeledObj, err := CreateResource(clusterConfig, c.typeConfig.GetTargetType(), targetObject)
+		if err != nil {
+			c.tl.Fatalf("Failed to create labeled resource in cluster %q: %v", clusterName, err)
+		}
+
+		clusterClient := genericclient.NewForConfigOrDie(clusterConfig)
+		defer func() {
```

Comment:
questioning a `defer` in a for loop

Reply:
good catch! This was part of an old change, it should not be a defer func.